### ISSUE TITLE
User can't add a message

### ISSUE THREAD

**Issue by codejamtc** [appirio-tech/connect-app#4040]
### Expected behavior
- User should be able to add a message

### Actual behavior
- User can't add a message

### Steps to reproduce the problem

- Login as Manager
- Go to Project listing 
- Click on a project - https://connect.topcoder-dev.com/projects/16567
- Go to 'Message'
- Click on start a new discussion and add the title and body text
- Click Post

### Screenshot/screencast

https://monosnap.com/file/R68Drfru4HWNJ8PJNM3JN0pPpxk2zh

#### Environment
**- OS:** HP 15-dw1xxx 15.6 inch; Window 10
**- Browser (w/version):** IE 11
**- User role (client, copilot or manager):** Manager
**- Account used:**  `TCConManager`

**Comment by vikasrohit** [comment]
@codejamtc I can see all the messages in the test project you used. It might the network that is causing slow loading of the messages or may be a network failure (timeout). Can you please check and share console logs, if you are still seeing such issue.

**Comment by maxceem** [comment]
@vikasrohit I think this issue is another variation of https://github.com/appirio-tech/connect-app/issues/4017 and https://github.com/appirio-tech/connect-app/issues/4029. In general: if we create a message with spaces only, then messages got broken.

**Comment by vikasrohit** [comment]
@maxceem but in this particular case, there was correct text in the messages and I am able to see them when I loaded those projects. So, it seems to be network issue to me.

**Comment by maxceem** [comment]
Right, it's different from the issues I've mentioned above. Though I can reproduce it https://monosnap.com/file/PZpDbX63OZDxKkeJ91jpBSjJJWSHRH. I can see messages previously created, but when I create a new one, I cannot see it.

Another thing, I've joined the project as a Manager, but in IE11 it doesn't show me even if I reload the page:

![image](https://user-images.githubusercontent.com/146016/82190737-29dfcc00-98fa-11ea-99da-ba3dc5e2212b.png)

But in Google Chrome it shows me as a Manager, and it shows new message:

![image](https://user-images.githubusercontent.com/146016/82190794-411eb980-98fa-11ea-8ccb-587899693da2.png)
